package.json

```diff
@@ -1,22 +1,37 @@
 {
   "name": "modzy-sdk",
-  "version": "1.0.1",
+  "version": "0.7.2",
```

Comment:
I'm guessing the intention here was to align this SDK version with our 0.7.2 release of Modzy, but we should never push a new version that has a lower version number otherwise it will never get pulled when someone runs `yarn install`.

Reply:
So the version bump should be `1.0.2` or something like that.